huaweicloud/services/rds/resource_huaweicloud_rds_pg_account.go

```diff
@@ -0,0 +1,344 @@
+// @Product RDS
+// ---------------------------------------------------------------
+
+package rds
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/pagination"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func ResourcePgAccount() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourcePgAccountCreate,
+		UpdateContext: resourcePgAccountUpdate,
+		ReadContext:   resourcePgAccountRead,
+		DeleteContext: resourcePgAccountDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"instance_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the ID of the RDS PostgreSQL instance.`,
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the username of the DB account.`,
+			},
+			"password": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `Specifies the password of the DB account.`,
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: `Specifies the remarks of the DB account.`,
+			},
+		},
+	}
+}
+
+func resourcePgAccountCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	cfg := meta.(*config.Config)
+	region := cfg.GetRegion(d)
+
+	// createPgAccount: create RDS PostgreSQL account.
+	var (
+		createPgAccountHttpUrl = "v3/{project_id}/instances/{instance_id}/db_user"
+		createPgAccountProduct = "rds"
+	)
+	createPgAccountClient, err := cfg.NewServiceClient(createPgAccountProduct, region)
+	if err != nil {
+		return diag.Errorf("error creating RDS client: %s", err)
+	}
+
+	instanceId := d.Get("instance_id").(string)
+	createPgAccountPath := createPgAccountClient.Endpoint + createPgAccountHttpUrl
+	createPgAccountPath = strings.ReplaceAll(createPgAccountPath, "{project_id}", createPgAccountClient.ProjectID)
+	createPgAccountPath = strings.ReplaceAll(createPgAccountPath, "{instance_id}", instanceId)
+
+	createPgAccountOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+	}
+
+	requestBody := buildCreatePgAccountBodyParams(d)
+	log.Printf("[DEBUG] Create RDS PostgreSQL account options: %#v", createPgAccountOpt)
+	requestBody["password"] = utils.ValueIngoreEmpty(d.Get("password"))
```

Comment:
```suggestion
```
set password in func: buildCreatePgAccountBodyParams

Reply:
no, the password should not be print in the log